yarn.lock

```diff
@@ -1214,6 +1214,53 @@
     tslib "^2.5.0"
     uuid "^8.3.2"
 
+"@aws-sdk/client-ssm@^3.441.0":
```

Comment:
Could we remove the `^` and fix the sdk version?

Reply:
And align it with the version we have across the repo: `3.421.0`.

Reply:
Updated!